lib/unhangout-routes.js

```diff
@@ -491,18 +549,51 @@ module.exports = {
             showEventOverview(req, res, "admin.ejs");
         });
 
-        app.get("/myevents/", ensureAuthenticated, function(req, res) {
+        app.get("/myevents/", ensureAuthenticated, utils.ensureAdmin,function(req, res) {
             showEventOverview(req, res, "myevents.ejs");
         });
 
+        app.post("/myevents/", ensureAuthenticated, utils.ensureAdmin, function(req, res) {
+
+            var eventID = req.body.eventId; 
+
+            var event = db.events.get(req.body.eventId);
```

Comment:
checking now if the user is an admin of the event before letting them continue https://github.com/unhangout/unhangout/commit/9fadf1ae26580300149df63f13cdcfcea4c1c31f